test/Microsoft.SqlTools.ServiceLayer.TestDriver/Tests/QueryExecutionTests.cs

```diff
@@ -323,5 +324,38 @@ public class QueryExecutionTests
                 await testHelper.Disconnect(queryTempFile.FilePath);
             }
         }
+
+        [Fact]
+        public async Task NoOpQueryReturnsMessage()
```

Comment:
We changed a while back to not send a result back if there was an error. Instead we would send an error back on the request context if there was an error. This did leave the message field behind as not necessary.